hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -10,6 +10,8 @@ parameters:
   required: true
 - name: CHANNEL
   value: staging
+- name: FEDRAMP
+  value: "${FEDRAMP}"
```

Comment:
This needs to be "false"

Reply:
https://github.com/openshift/certman-operator/pull/209 opened to address this